src/include/duckdb.h

```diff
@@ -1710,6 +1710,16 @@ Adds a parameter to the table function.
 */
 DUCKDB_API void duckdb_table_function_add_parameter(duckdb_table_function table_function, duckdb_logical_type type);
 
+/*!
+Adds a named parameter to the table function.
+
+* table_function: The table function
+* name: The name of the parameter
+* type: The type of the parameter to add.
+*/
+DUCKDB_API void duckdb_table_function_add_named_parameter(duckdb_table_function table_function, const char *name,
```

Comment:
Should there not also be a corresponding `duckdb_bind_get_named_parameter` or similar to allow access to these parameters during the bind phase?

Reply:
Yes, you're right. Have that in a local branch and will push up.

Reply:
Added `duckdb_bind_get_named_parameter`